### PR TITLE
COST-2106: Remove unnecessary GCP partables.

### PR DESCRIPTION
**Pull request by myersCody**
In an effort to do only additive changes I am gonna do this issue in two prs. The [first pull request](https://github.com/project-koku/koku/pull/3400) removed the unnecessary GCP partables from the provider map. When no endpoint in production can attempt to pull data from these tables, we can merge in the 2nd pull request (this one) to delete the tables. 

## How to test
**Step One:**
On the main branch load some gcp data and check that the unnecessary GCP partables exist.

*SQL*
```
select table_schema,
       table_name
  from information_schema.tables
 where table_name in (
    'reporting_gcp_compute_summary_by_project_p',
    'reporting_gcp_compute_summary_by_service_p',
    'reporting_gcp_compute_summary_by_region_p',
       );
```
*Expected Result*
```
 table_schema |                 table_name
--------------+--------------------------------------------
 template0    | reporting_gcp_compute_summary_by_service_p
 template0    | reporting_gcp_compute_summary_by_region_p
 template0    | reporting_gcp_compute_summary_by_project_p
 acct10001    | reporting_gcp_compute_summary_by_service_p
 acct10001    | reporting_gcp_compute_summary_by_region_p
 acct10001    | reporting_gcp_compute_summary_by_project_p
 (12 rows)
 ```
 **Step Two**
 Switch over to this branch, and run the migration.
 ```
 make run-migrations
 ```
 **Step Three*
 Log back into the db, and check to make sure the unnecessary tables don't exit anymore. 
 *Expected Result*
 ```
  table_schema | table_name
--------------+------------
(0 rows)
 ```

### CODE REVIEW

**Comment by myersCody** [comment]
Blocked by: https://github.com/project-koku/koku/pull/3408

**Review by Red-HAP — APPROVED** [review]

File: koku/reporting/provider/gcp/models.py
Comment (maskarb) [inline koku/reporting/provider/gcp/models.py:25]: Let's move the deletions from `UI_SUMMARY_TABLES` to a separate PR and ship to prod before merging these migrations.

**Review by maskarb — CHANGES_REQUESTED** [review]

**Review by maskarb — APPROVED** [review]

**Review by maskarb — APPROVED** [review]

**Pull request merged**
